source/features/previous-version.tsx

```diff
@@ -0,0 +1,126 @@
+import cache from 'webext-storage-cache';
+import React from 'dom-chef';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+import * as api from '../github-helpers/api';
+import GitHubURL from '../github-helpers/github-url';
+
+const getPastCommits = cache.function('previous-version', async (currHref: string): Promise<string[] | false> => {
+	const githubUrl = new GitHubURL(currHref);
+	const {resource: {history}} = await api.v4(`
+		resource(url: "/${githubUrl.user}/${githubUrl.repository}/commit/${githubUrl.branch}") {
+			... on Commit {
+				history(path: "${githubUrl.filePath}", first: 5) {
+					nodes {
+						oid
+					}
+				}
+			}
+		}
+	`);
+
+	const nodes = history.nodes as any[];
+
+	if (nodes.length === 0) {
+		return false;
+	}
+
+	return nodes.map(node => node.oid as string);
```

Comment:
```suggestion
	return nodes.slice(1).map(node => node.oid as string);
```


If the first commit is always the current commit, just slice it here so you don't have to use [1] and .slice(2) later on